ice.c

```diff
@@ -1669,6 +1709,9 @@ static void janus_ice_component_free(const janus_refcount *component_ref) {
 		janus_seq_list_free(&component->last_seqs_video[1]);
 	if(component->last_seqs_video[2])
 		janus_seq_list_free(&component->last_seqs_video[2]);
+	int i = 0;
+	for(i=0; i<JANUS_MAX_PENDING_MESSAGES; i++)
```

Comment:
nit: no need to initialize i twice.

Reply:
Makes sense :smile:  The paranoid in me tends to initialize everything, these days, as I've had my share of issues caused by a lack of that.